src/static/getConfig.js

```diff
@@ -212,6 +212,11 @@ export const buildConfigation = (config = {}) => {
   process.env.REACT_STATIC_PREFETCH_RATE = finalConfig.prefetchRate
   process.env.REACT_STATIC_DISABLE_ROUTE_INFO_WARNING = finalConfig.disableRouteInfoWarning
   process.env.REACT_STATIC_DISABLE_ROUTE_PREFIXING = finalConfig.disableRoutePrefixing
+  process.env.REACT_STATIC_BUILD_HASH =
+    finalConfig.buildHash ||
+    Math.random()
+      .toString(36)
+      .substr(2, 9)
```

Comment:
Would you rather use something like the "shortid" package for this?

Reply:
Done!